### PR TITLE
[MI-2307] Update readme files

### PR DESCRIPTION
**Pull request by ayusht2810**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,14 @@ This plugin contains the following features:
 - Notifications will be sent in the form of a post created by the ServiceNow bot in the channel specified while creating the subscription.
 - Ability to see the existing subscriptions in the Right Hand Sidebar or by using a slash command.
 - Ability to delete the subscriptions from the Right Hand Sidebar or by using a slash command.
+- Ability to filter subscriptions using the slash command to get a post containing filtered subscriptions.
+- Ability to filter subscriptions in the Right Hand Sidebar using the filter icon.
+- Search and share a ServiceNow record in a specified channel.
+- Ability to open search and share record modal through UI or by using a slash command.
+- Add and view comments of a ServiceNow record.
+- Update the state of a ServiceNow record.
+- Ability to open add and view comments/update state modal through buttons present in a notification post or a shared record post.
```
Comment (raghavaggarwal2308) [inline README.md:43]:
```suggestion
- Ability to open, add and view comments/update state modal through buttons present in a notification post or a shared record post.
```
Also, `view comments/update state modal` is sounding a little weird

File: README.md [hunk 2]
```diff
@@ -34,6 +34,14 @@ This plugin contains the following features:
...
+- Supported record types for sharing a record or updating a record state - incident, problem, change_request, kb_knowledge, task, change_task, cert_follow_on_task.
```
Comment (raghavaggarwal2308) [inline README.md:44]:
```suggestion
- Supported record types for sharing a record or updating a record state - incident, problem, change_request, kb_knowledge, task, change_task and cert_follow_on_task.
```

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -34,6 +34,14 @@ This plugin contains the following features:
 - Notifications will be sent in the form of a post created by the ServiceNow bot in the channel specified while creating the subscription.
 - Ability to see the existing subscriptions in the Right Hand Sidebar or by using a slash command.
 - Ability to delete the subscriptions from the Right Hand Sidebar or by using a slash command.
+- Ability to filter subscriptions using the slash command to get a post containing filtered subscriptions.
+- Ability to filter subscriptions in the Right Hand Sidebar using the filter icon.
```
Comment (Kshitij-Katiyar) [inline README.md:38]:
```suggestion
- Ability to filter subscriptions in the Right-Hand Sidebar using the filter icon.
```

File: docs/developer_docs.md [hunk 4]
```diff
@@ -37,6 +37,14 @@ This plugin contains the following features:
 - Notifications will be sent in the form of a post created by the ServiceNow bot in the channel specified while creating the subscription.
 - Ability to see the existing subscriptions in the Right Hand Sidebar or by using a slash command.
 - Ability to delete the subscriptions from the Right Hand Sidebar or by using a slash command.
+- Ability to filter subscriptions using the slash command to get a post containing filtered subscriptions.
+- Ability to filter subscriptions in the Right Hand Sidebar using the filter icon.
```
Comment (Kshitij-Katiyar) [inline docs/developer_docs.md:41]:
```suggestion
- Ability to filter subscriptions in the Right-Hand Sidebar using the filter icon.
```

File: docs/developer_docs.md [hunk 5]
```diff
@@ -37,6 +37,14 @@ This plugin contains the following features:
...
+- Search and share a ServiceNow record in a specified channel.
+- Ability to open search and share record modal through UI or by using a slash command.
```
Comment (Kshitij-Katiyar) [inline docs/developer_docs.md:43]:
```suggestion
- Ability to open search and share record modal through UI or slash command.
```

File: README.md (hunk 1 above)
Comment (ayusht2810) [reply README.md]: updated the sentence

**Review by Kshitij-Katiyar — APPROVED** [review]

**Review by raghavaggarwal2308 — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -32,8 +32,16 @@ This plugin contains the following features:
...
+- Ability to see the existing subscriptions in the Right-Hand Sidebar or slash command.
+- Ability to delete the subscriptions from the Right-Hand Sidebar or slash command.
+- Ability to filter subscriptions using the slash command to get a post containing filtered subscriptions.
+- Ability to filter subscriptions in the Right-Hand Sidebar using the filter icon.
+- Search and share a ServiceNow record in a specified channel.
+- Ability to open search and share record modal through UI or slash command.
+- Add and view comments of a ServiceNow record.
+- Update the state of a ServiceNow record.
+- Ability to open, add and view comments modal or update state modal through buttons present in a notification post or a shared record post.
+- Supported record types for sharing a record or updating a record state - incident, problem, change_request, kb_knowledge, task, change_task and cert_follow_on_task.
```
[the remark this answers is not in the document]
Comment (ayusht2810) [reply README.md]: updated types

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**